models/models.go

```diff
@@ -851,22 +851,26 @@ func (m *RouteManager) GetDNSInstructions(route *Route) ([]string, error) {
 		"instance-id": route.InstanceId,
 	})
 
+	lsession.Info("load-user")
 	user, err := route.loadUser(m.db)
 	if err != nil {
-		lsession.Error("load-user", err)
+		lsession.Error("load-user-err", err)
 		return instructions, err
 	}
 
 	if err := json.Unmarshal(route.ChallengeJSON, &challenges); err != nil {
 		lsession.Error("json-unmarshal-challenge", err)
 		return instructions, err
 	}
+
+	lsession.Info("get-key-authorization")
 	for _, auth := range challenges {
 		for _, challenge := range auth.Body.Challenges {
 			if challenge.Type == acme.DNS01 {
+				lsession.Info("get-key-authorization-for-a-dns-challenge", lager.Data{"domain": auth.Domain})
 				keyAuth, err := acme.GetKeyAuthorization(challenge.Token, user.GetPrivateKey())
 				if err != nil {
-					lsession.Error("get-key-authorization", err)
+					lsession.Error("get-key-authorization-err", err)
```

Comment:
Is it possible to include the domain in the data here too? I know these logs can sometimes be a bit dense and hard to parse, so having the information in a line single is helpful.

Reply:
Resolved by including the domains in all this function via the logger.Session call.